backend/tests/mw_drw_proxy/proxy.py

```diff
@@ -0,0 +1,165 @@
+import json
+import os
+import typing
+
+from flask import Flask, request
+import logging
+import traceback
+
+from requests import HTTPError
+
+from tests.e2e_tests import UserClient
+
+APP_NAME = __name__.split(".")[0]
+LOG = logging.getLogger(APP_NAME)
+logging.basicConfig(level=logging.INFO)
+
+app = Flask(APP_NAME)
+
+STATE: typing.Dict[str, UserClient] = {}
+
+DRW_URL_PREFIX = os.getenv("DRW_URL_PREFIX", "https://demo-wallet.diem.com")
+
+
+def error_response(e):
+    status_code = 400
+    if isinstance(e, HTTPError):
+        status_code = e.response.status_code
+    return {
+        "error": str(e),
+        "stacktrace": traceback.format_tb(e.__traceback__),
+    }, status_code
+
+
+@app.route("/reset", methods=["GET", "POST"])
+def reset_test():
+    STATE.clear()
+
+
+@app.route("/accounts", methods=["POST"])
+def create_account():
+    try:
+        data = request.json
+
+        # "{\"type\": \"individual\", \"payload_version\": 1, \"given_name\": \"Tom\", \"surname\": \"Jack\"}"
+        kyc_data = json.loads(data["kyc_data"])
+        kyc_info = {
+            "first_name": kyc_data["given_name"],
+            "last_name": kyc_data["surname"],
+            "dob": "1956-07-09",
+            "phone": "1 202-456-1414",
+            "address_1": "1840 Century Park East",
+            "address_2": "",
+            "city": "Los Angeles",
+            "state": "California",
+            "zip": "90067",
+            "country": "US",
+            "selected_fiat_currency": "USD",
+            "selected_language": "en",
+        }
+        uc = UserClient.create(
+            DRW_URL_PREFIX,
+            "_".join([kyc_data["given_name"], kyc_data["surname"]]).lower(),
+            kyc_info=kyc_info,
+            logger=logging.info,
+        )
+
+        for currency, amount in data["balances"].items():
+            if amount > 0:
+                uc.buy(amount, currency, "USD")
+
+        STATE[uc.name] = uc
```

Comment:
I probably will change STATE to a list, then use len(STATE) as id, so that we are sure no duplicate id created.  Depending on name seems easy to get conflict id.

Reply:
I like using the full usernames as it makes it easier during debugging to log in and less looking up multiple values in different places, but I added a function to ensure username uniqueness